scripts/debug

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -e
+
+# This script is based on raspinfo: https://github.com/raspberrypi/utils/blob/master/raspinfo/raspinfo
+# raspibfo is Copyright: 2018-2019, Raspberry Pi (Trading) Ltd.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the University nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
+# A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR 
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# This fork is Copyright: 2020 Umbrel.
+
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+
+echo "====================="
+echo "= Umbrel debug info ="
+echo "====================="
+
+UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+cd "${UMBREL_ROOT}"
+echo
+echo "Umbrel version"
+echo "---------------"
+cat info.json | jq .version | sed 's/"//g'
+
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+    echo
+    echo "Umbrel OS version"
+    echo "-----------------"
+    echo $UMBREL_OS
+
+    echo
+    echo "Raspberry Pi Model"
+    echo "------------------"
+
+    cat /proc/cpuinfo | tail -3
+
+    echo
+    echo "Firmware"
+    echo "--------"
+
+    vcgencmd version
+
+    echo
+    echo "Tempature"
+    echo "---------"
+
+    vcgencmd measure_temp
+
+    echo
+    echo "Throttling"
+    echo "----------"
+
+    vcgencmd get_throttled
+
+    echo
+    echo "Startup status"
+    echo "--------------"
+    systemctl status umbrel-startup
```

Comment:
Since `systemctl` pages the logs, it'd be easier to fetch the last 30 lines via `journalctl`:

```suggestion
    journalctl --unit=umbrel-startup.service | tail -n 30
```